src/index.ts

```diff
@@ -136,6 +137,10 @@ export default function app(): FastifyInstance {
   });
 
   app.register(helmet);
+  app.register(cors, {
+    origin: isProd ? /^(?:https:\/\/)?(?:[\w-]+\.)*daily\.dev$/ : true,
```

Comment:
Little modified regex vs what we have currently in API `/daily\.dev$/` because it allows origins like `adaily.dev` which is invalid. Will update on API as well in https://github.com/dailydotdev/daily-api/pull/2048